cost-analyzer/templates/forecasting-service-account.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.forecasting.enabled }}
+{{- if and .Values.serviceAccount.create .Values.forecasting.serviceAccountName }}
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
I don't think we need this file? @jessegoodier Any reason you created it in the initial draft?

Reply:
Good point Thomas, I removed this. If it causes problems it can be reintroduced.